controllers/utils/conditions.go

```diff
@@ -32,43 +32,45 @@ type ConditionReason string
-	InProgress                 ConditionReason
-	InvalidPlatformImage       ConditionReason
-	MissingBlockingCR          ConditionReason
-	NotAllManagedPoliciesExist ConditionReason
-	NotEnabled                 ConditionReason
-	NotStarted                 ConditionReason
-	ClusterNotFound            ConditionReason
-	NotPresent                 ConditionReason
-	PartiallyDone              ConditionReason
-	PrecacheSpecIsWellFormed   ConditionReason
-	TimedOut                   ConditionReason
+	Completed                     ConditionReason
+	ClusterSelectionCompleted     ConditionReason
+	ValidationCompleted           ConditionReason
+	BackupCompleted               ConditionReason
+	PrecachingCompleted           ConditionReason
+	Failed                        ConditionReason
+	IncompleteBlockingCR          ConditionReason
+	InProgress                    ConditionReason
+	InvalidPlatformImage          ConditionReason
+	MissingBlockingCR             ConditionReason
+	NotAllManagedPoliciesExist    ConditionReason
+	AmbiguousManagedPoliciesNames ConditionReason
+	NotEnabled                    ConditionReason
+	NotStarted                    ConditionReason
+	ClusterNotFound               ConditionReason
+	NotPresent                    ConditionReason
+	PartiallyDone                 ConditionReason
+	PrecacheSpecIsWellFormed      ConditionReason
+	TimedOut                      ConditionReason
 }{
-	Completed:                  "Completed",
-	ClusterSelectionCompleted:  "ClusterSelectionCompleted",
-	ValidationCompleted:        "ValidationCompleted",
-	BackupCompleted:            "BackupCompleted",
-	PrecachingCompleted:        "PrecachingCompleted",
-	Failed:                     "Failed",
-	IncompleteBlockingCR:       "IncompleteBlockingCR",
-	InProgress:                 "InProgress",
-	InvalidPlatformImage:       "InvalidPlatformImage",
-	MissingBlockingCR:          "MissingBlockingCR",
-	NotAllManagedPoliciesExist: "NotAllManagedPoliciesExist",
-	NotEnabled:                 "NotEnabled",
-	NotStarted:                 "NotStarted",
-	ClusterNotFound:            "ClusterNotFound",
-	NotPresent:                 "NotPresent",
-	PartiallyDone:              "PartiallyDone",
-	PrecacheSpecIsWellFormed:   "PrecacheSpecIsWellFormed",
-	TimedOut:                   "TimedOut",
+	Completed:                     "Completed",
+	ClusterSelectionCompleted:     "ClusterSelectionCompleted",
+	ValidationCompleted:           "ValidationCompleted",
+	BackupCompleted:               "BackupCompleted",
+	PrecachingCompleted:           "PrecachingCompleted",
+	Failed:                        "Failed",
+	IncompleteBlockingCR:          "IncompleteBlockingCR",
+	InProgress:                    "InProgress",
+	InvalidPlatformImage:          "InvalidPlatformImage",
+	MissingBlockingCR:             "MissingBlockingCR",
+	NotAllManagedPoliciesExist:    "NotAllManagedPoliciesExist",
+	AmbiguousManagedPoliciesNames: "AmbiguousManagedPoliciesNames",
```

Comment:
Should it be `AmbiguousManagedPolicyNames`?

Reply:
The initial commit has it the same as in here. I did a cherry-pick without changing: https://github.com/openshift-kni/cluster-group-upgrades-operator/commit/499ad67a462aade819b70cfbfb98e7b56be02dc3#diff-9dc0c6962ba7ab03a8510d5d877f487287f4517b59d757ddd02544008a6dcb74R46

Reply:
Right, we'll have to do another PR in main anyways if we decide to update the name.